STO ($SrTiO_3$) substrate, $a = 3.905 $A, mass density: $\rho = 5.122 g/cm^3$, 

Bragg formula:

$$ 2d \sin \theta = n \lambda$$

![Bragg Formula](Bragg.png)

X-ray Cu K-$\alpha$ line, wave length: $\lambda = 1.5405952 $ A (8.04 kV)

$$\theta = \sin^{-1} \frac{\lambda }{2 d}$$

use: $\frac{\pi}{180} = \frac{\theta}{\text{deg}}$, to convert $\theta $ to degrees.

$$ \text{deg} = \frac{180 \times \theta}{\pi} $$

In [5]:
#!/usr/bin/python
import math

Lambda = 1.5405952
d = 3.905

theta = math.asin (Lambda/(2*d))

print ("Angle (100) direction in arc: ", theta)

degree = 180*theta/math.pi

print ("Omega: ", degree)

print ("2Theta: ", 2*degree)

Angle (100) direction in arc:  0.1985615087939002
Omega:  11.376736427640264
2Theta:  22.753472855280528


Separation between Lattice Planes in a cubic crystal, (h,k,l) direction: 

$$d_{h,k,l} = \frac{a }{\sqrt{h^2 + k^2 + l^2}}$$

Separation between (200) planes:

$$d = \frac{a}{2} $$

(For $2 a \sin \theta = n \lambda$, $n = 2$)

In [6]:
Lambda = 1.5405952
d = 3.905/2.

theta = math.asin (Lambda/(2*d))

print ("Angle (200) direction in arc: ", theta)

degree = 180*theta/math.pi

print ("Omega: ", degree)

print ("2Theta: ", 2*degree)

Angle (200) direction in arc:  0.40554391664868134
Omega:  23.23595483117468
2Theta:  46.47190966234936


在XRD中，对STO(100)取向的衬底，设置2Theta为：46.48, Omega为：23.24。

考虑STO(111)方向，

$$d_{111} = \frac{a}{\sqrt{3}} $$

In [8]:
Lambda = 1.5405952
d = 3.905/math.sqrt(3.)

theta = math.asin (Lambda/(2*d))

print ("Angle (111) direction in arc: ", theta)

degree = 180*theta/math.pi

print ("Omega: ", degree)

print ("2Theta: ", 2*degree)

Angle (111) direction in arc:  0.3486859666759119
Omega:  19.978234265969018
2Theta:  39.956468531938036


对STO(111)取向的衬底，我们一般取2Theta：40，Omega: 20。

Consider STO(110) cut substrate, we have:

$$d_{110} =\frac{a}{\sqrt 2} $$

In [9]:
Lambda = 1.5405952
d = 3.905/math.sqrt(2.)

theta = math.asin (Lambda/(2*d))

print ("Angle (110) direction in arc: ", theta)

degree = 180*theta/math.pi

print ("Omega: ", degree)

print ("2Theta: ", 2*degree)

Angle (110) direction in arc:  0.28271801693067566
Omega:  16.198549162435867
2Theta:  32.397098324871735


Substrate | cut | 2Theta | Omega
--- | --- | --- | ---
STO | 200 | 46.48 | 23.24 
STO | 111 | 39.96 | 19.98
STO | 110 | 32.40 | 16.20